runtime/src/bank.rs

```diff
@@ -1813,7 +1813,11 @@ impl Bank {
 
     /// computed unix_timestamp at this slot height
     pub fn unix_timestamp_from_genesis(&self) -> i64 {
-        self.genesis_creation_time + ((self.slot as u128 * self.ns_per_slot) / 1_000_000_000) as i64
+        self.genesis_creation_time.saturating_add(
+            (self.slot as u128)
+                .saturating_mul(self.ns_per_slot)
+                .saturating_div(1_000_000_000) as i64,
+        )
```

Comment:
(nit: actually, I'd like to see use of checked math here instead, by setting `self.ns_per_slot` to `u64::MAX as u128` in `::new_no_wallclock_throttle_for_tests()` as an idealist... But, at the other hand, i question those efforts are really worthwhile... :p)

Reply:
the whole thing kind of seems useless to me; it's not going to be anywhere near the real unix timestamp because our slot time drifts.